### DISCUSSION TITLE
Depoly clean data base, but continue payouts for existing shares.

### DISCUSSION THREAD

**Discussion by hots82** [coinfoundry/miningcore#1307] [Q&A] [answered]
Hello, here...
Theoretically... If i run another instance of miningcore on a separate machine, and import existing database, then connect it to same daemon. It should process payments for existing shares? 
I want to deploy clean database but want to complete payments for existing shares...

**Comment by oliverw** [comment, marked as answer]
Yes.

**Reply by hots82** [reply]
> Yes.

Thanks... Another quick question... I've changed a name for pool in ```config.json```. But used same database. Will it process existing recorded shares?

**Reply by oliverw** [reply]
Changing the pool's `id` in config.json won't work for this scenario. The ids must match. You can change it after the payouts have been fully processed.

**Reply by hots82** [reply]
> Changing the pool's `id` in config.json won't work for this scenario. The ids must match. You can change it after the payouts have been fully processed.

I already did..  Did I screwed it? Or using existing database with another instance with matched pool `id` will work?
Sorry for noobs questions. I have no knowledge about databases at all.

**Reply by oliverw** [reply]
> > Changing the pool's `id` in config.json won't work for this scenario. The ids must match. You can change it after the payouts have been fully processed.
> 
> I already did.. Did I screwed it? Or using existing database with another instance with matched pool `id` will work? Sorry for noobs questions. I have no knowledge about databases at all.

Just make the id in the config match the id in the database and restart the pool. No damage done.
